tests/components/nexia/util.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for the nexia integration."""
+import uuid
+
+from asynctest import patch
+from nexia.home import NexiaHome
+import requests_mock
+
+from homeassistant.components.nexia.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, load_fixture
+
+
+async def async_init_integration(
+    hass: HomeAssistant, skip_setup: bool = False,
+) -> MockConfigEntry:
+    """Set up the nexia integration in Home Assistant."""
+
+    house_fixture = "nexia/mobile_houses_123456.json"
+    session_fixture = "nexia/session_123456.json"
+    sign_in_fixture = "nexia/sign_in.json"
+
+    with requests_mock.mock() as m, patch(
```

Comment:
`requests_mock` exists as a pytest fixture that we can provide as an argument to the test. We'll save some indentation here.

Reply:
I see now that there are two patches. So we won't save indentation directly.